src/runtimes/go.js

```diff
@@ -1,8 +1,38 @@
-const { join } = require('path')
+const { lstat } = require('fs')
+const { join, extname, dirname, basename } = require('path')
+const { promisify } = require('util')
+
+const pLstat = promisify(lstat)
 
 const cpFile = require('cp-file')
 
-const { zipBinary } = require('../runtime')
+const { RUNTIME_GO } = require('../utils/consts')
+const { zipBinary } = require('../zip_binary')
+
+const { detectBinaryRuntime } = require('./detect_runtime')
+
+const findFunctionsInPaths = async function (paths) {
```

Comment:
This function is pretty similar from the Rust one, should those share some common code?

Reply:
Sounds good :+1: